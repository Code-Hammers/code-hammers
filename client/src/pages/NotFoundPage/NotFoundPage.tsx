import React from 'react'

const NotFoundPage = (): JSX.Element => {
  return (
    <div>NotFoundPage</div>
  )
}

export default NotFoundPage