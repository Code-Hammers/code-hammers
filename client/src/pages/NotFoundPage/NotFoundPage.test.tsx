import { create } from 'react-test-renderer';

import NotFoundPage from './NotFoundPage';

describe('MainPage Component', () => {
  it('renders correctly', () => {
    const tree = create(<NotFoundPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
