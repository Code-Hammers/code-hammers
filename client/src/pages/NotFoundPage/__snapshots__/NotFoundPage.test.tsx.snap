// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage Component renders correctly 1`] = `
<div>
  NotFoundPage
</div>
`;
